tests/jballerina-integration-test/src/test/resources/transaction/src/participantservice/01_remote_participant.bal

```diff
@@ -375,6 +375,45 @@ function remoteErrorReturnInitiator() returns @tainted string {
     return s;
 }
 
+
+function callParticipantMultipleTimes() returns string {
```

Comment:
This testcase will be used to check multiple execution of same local and remote participants